packages/types/src/organization.ts

```diff
@@ -62,6 +62,7 @@ export interface UpdateMembershipParams {
 
 export interface UpdateOrganizationParams {
   name: string;
+  slug: string;
```

Comment:
🔧 
Let's make it optional, otherwise this is a breaking change as we only required `name` until now.
Is this required by FAPI?

Reply:
@nikosdouvlis i don’t think that it is mandatory